src/dune/dep_conf_eval.mli

```diff
@@ -0,0 +1,13 @@
+(** Interpret dependencies written in Dune files *)
+open Stdune
+
+(** Evaluates to the actual list of dependencies, ignoring aliases, and
```

Comment:
Why ignoring aliases? The implementation seems to handle them.

Reply:
Not sure what I meant by that comment. Will fix.